### PR TITLE
fixed unknown fields in config/jobs/kubernetes-sigs/kind/kind.yaml

### PR DESCRIPTION
**Pull request by asim-reza**
### Issue
[error unmarshaling config/jobs/kubernetes-sigs/kind/kind.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"always_run\" #27454](https://github.com/kubernetes/test-infra/issues/27454)

As discussed in the aforementioned issue.

### CODE REVIEW

**Comment by aojea** [comment]
/lgtm

Ben for approval, the renaming makes sense to me

**Comment by aojea** [comment]
Thanks @asim-reza , the job seems to fail on others jobs with issues

**Comment by asim-reza** [comment]
> Thanks @asim-reza , the job seems to fail on others jobs with issues

Yea all the issues need to be fixed for it to pass... I have opened issues for all... it should pass once all merge... The script https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-csi/gen-jobs.sh needs to be fixed as well as it creates these issues... Im working on that currently... I'll create an issue for that with the fixes soon ✌🏼

**Comment by aojea** [comment]
yeah, that helps reviewers to understand better the changes and we minimize the risk of introducing problems

**Comment by asim-reza** [comment]
> yeah, that helps reviewers to understand better the changes and we minimize the risk of introducing problems

Now the question is will it merge with the job failing? (it was set to be optional)

**Comment by aojea** [comment]
> Now the question is will it merge with the job failing? (it was set to be optional)

let's use this as canary

/approve

File: config/jobs/kubernetes-sigs/kind/kind.yaml [hunk 1]
```diff
@@ -20,11 +19,11 @@ periodics:
...
       securityContext:
         privileged: true
-    annotations:
-      testgrid-dashboards: sig-testing-kind
-      testgrid-tab-name: ci unit test
-      testgrid-alert-email: bentheelder@google.com
-      description: kind CI unit test
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: kind-ci-unit-test
```
Comment (aojea) [inline config/jobs/kubernetes-sigs/kind/kind.yaml:24]: what has this changed?

Comment (asim-reza) [reply config/jobs/kubernetes-sigs/kind/kind.yaml:24]: Kindly reference the [issue](https://github.com/kubernetes/test-infra/issues/27454) linked above for the diff and details as to why

Comment (asim-reza) [reply config/jobs/kubernetes-sigs/kind/kind.yaml:24]: TLDR indentation was changed...

File: config/jobs/kubernetes-sigs/kind/kind.yaml [hunk 2]
```diff
@@ -1,13 +1,12 @@
 periodics:
-- name: ci-kind-test
+- name: ci-kind-unit-test
```
Comment (aojea) [inline config/jobs/kubernetes-sigs/kind/kind.yaml:2]: can we keep the old name?

**Comment by aojea** [comment]
/hold cancel

It seems the job name is duplicated on another job, the rename makes sense to me

File: config/jobs/kubernetes-sigs/kind/kind.yaml (hunk 2 above)
Comment (aojea) [reply config/jobs/kubernetes-sigs/kind/kind.yaml:2]: no need

File: config/jobs/kubernetes-sigs/kind/kind.yaml (hunk 1 above)
Comment (aojea) [reply config/jobs/kubernetes-sigs/kind/kind.yaml:24]: no need

Comment (aojea) [reply config/jobs/kubernetes-sigs/kind/kind.yaml:24]: I meant the name , it was `ci unit test`

**Comment by aojea** [comment]
/lgtm

unit test fails without renaming

**Pull request merged**